src/sssom/context.py

```diff
@@ -25,108 +22,79 @@
 )
 
 
-def get_jsonld_context():
-    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file.
-
-    [Auto generated from sssom.yaml by jsonldcontextgen.py]
-
-    :return: JSON-LD context
-    """
+def _get_jsonld_context():
+    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file."""
     with open(SSSOM_CONTEXT, "r") as c:
         context = json.load(c, strict=False)
-
     return context
 
 
-def get_extended_prefix_map():
-    """Get prefix map from bioregistry (obo.epm.json).
-
-    :return: Prefix map.
-    """
-    converter = Converter.from_extended_prefix_map(EXTENDED_PREFIX_MAP)
-    return {record.prefix: record.uri_prefix for record in converter.records}
+def get_internal_converter() -> Converter:
+    """Get a converter from the SSSOM internal context."""
+    context = _get_jsonld_context()
+    prefix_map = {}
+    for key in context["@context"]:
+        v = context["@context"][key]
+        if isinstance(v, str):
+            prefix_map[key] = v
+        elif isinstance(v, dict):
+            if "@id" in v and "@prefix" in v:
+                if v["@prefix"]:
+                    prefix_map[key] = v["@id"]
+    del prefix_map["@vocab"]
+    return Converter.from_prefix_map(prefix_map)
 
 
-def get_built_in_prefix_map() -> PrefixMap:
-    """Get built-in prefix map from the sssom_context variable in the auto-generated 'internal_context.py' file.
+def get_external_converter() -> Converter:
```

Comment:
we only ned this from now on, so "external" / "internal" dichotomy is superfluous.